velox/functions/sparksql/Register.cpp

```diff
@@ -78,6 +79,7 @@ static void workAroundRegistrationMacro(const std::string& prefix) {
   VELOX_REGISTER_VECTOR_FUNCTION(udf_not, prefix + "not");
   registerIsNullFunction(prefix + "isnull");
   registerIsNotNullFunction(prefix + "isnotnull");
+  registerFunction<IsNanFunction, bool, float>({prefix + "is_nan"});
```

Comment:
Do we need to register double type as well for Spark? If so, this function does not depend on Presto's registration. Thanks.

Reply:
Make sense, updated.

Reply:
Thanks. I have no further comment.